### PR TITLE
Remove unneeded nagios plugin dir attribute

### PR DESCRIPTION
**Pull request by rettigs**
Since the Nagios cookbook provides it.  I'm not sure why this ever existed in the first place.

### CODE REVIEW

**Comment by armiller** [comment]
:+1:

**Comment by ramereth** [comment]
@rettigs can you fix ``.kitchen.cloud.yml`` so that ``OS_TENANT`` is ``OS_TENANT_NAME``? And also update to using CentOS 6.6. That way I can run this on OpenStack.

**Comment by armiller** [comment]
:+1: LGTM

**Comment by DesertBeagle** [comment]
Is this still up to date?

**Comment by ramereth** [comment]
@DesertBeagle this needs to be rebased.

**Comment by DesertBeagle** [comment]
I rebased this and removed block line count limits, but it only fails with Jenkins. I have it working on my workstation.

File: .chef/knife.rb [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-client_key "#{current_dir}/fakeclient.pem"
-validation_client_name 'chef-validator'
-validation_key "#{current_dir}/validator.pem"
-chef_server_url 'https://api.opscode.com/organizations/my_awesome_org'
-cache_type 'BasicFile'
-cache_options(path: "#{ENV['HOME']}/.chef/checksums")
-cookbook_path ["#{current_dir}/../cookbooks"]
-role_path "#{current_dir}/../test/integration/roles"
-data_bag_path "#{current_dir}/../test/integration/data_bags"
-knife[:secret_file] = "#{current_dir}../test/integration/encrypted_data_bag_secret"
```
Comment (ramereth) [inline .chef/knife.rb]: Why is this removed? This is needed for chef-provisioning testing.

Comment (DesertBeagle) [inline .chef/knife.rb]: What is chef-provisioning testing?

Comment (ramereth) [inline .chef/knife.rb]: It allows us to do multi-node testing which test-kitchen doesn't have support for. Please keep it in here.

**Comment by DesertBeagle** [comment]
Added knife.rb back.

**Comment by ramereth** [comment]
@DesertBeagle can you please fix the rspec tests so they pass? You need to change CentOS 6.8 to 6.7 in spec_helper.rb

**Comment by DesertBeagle** [comment]
@ramereth Made the change!

**Comment by DesertBeagle** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.0.7.  Have a nice day!
